applications/launchpad_v2/src/utils/ExpertViewUtils.ts

```diff
@@ -0,0 +1,41 @@
+import { useSpring } from 'react-spring'
+import { ExpertViewType } from '../store/app/types'
+
+/**
+ * Helpers for the ExpertView
+ */
+class ExpertViewUtils {
+  /**
+   * Helper used to convert 'hidden', 'open', 'fullscreen' into actual '%' value
+   * @param {ExpertViewType} expertView
+   * @param {string} [openViewSize = '50%'] - the value for 'open' mode, ie. '40%'
+   */
+  static convertExpertViewModeToValue = (
+    expertView: ExpertViewType,
+    openViewSize = '50%',
+  ) => {
+    let size = '0%'
+    switch (expertView) {
+      case 'open':
+        size = openViewSize
+        break
+      case 'fullscreen':
+        size = '100%'
+        break
+      default:
+        size = '0%'
+        break
+    }
+
+    const invertedSize = `${100 - parseFloat(size)}%`
```

Comment:
you wouldn't have to parse if you used numbers and only returned `${size}%` at the end

Reply:
yeah, but this is an implementation detail, you can return it from here with `%`, but do calculations on numbers to avoid parse, but I understand your reasoning, so you can skip